### PR TITLE
[for REL-v7.0] Add support for Aosbox (H3ULCB ES3.0 8GB) 

### PR DESCRIPTION
**Pull request by varder**
Add support for H3ULCB AB ES3.0 8GB machine(CCPF Starter kit).
The configuration is based on R-Car Starter Kit H3ULCB 4x2g.

https://github.com/xen-troops/meta-xt-images/pull/209

### CODE REVIEW

**Comment by otyshchenko1** [comment]
lgtm, but please add [for REL-v7.0] prefix to the PR's subject to avoid confusion as we have two PR with the same subject.

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-4x2g-ab-dom0.dts [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+	iommus = <&ipmmu_ds1 32>;
+};
+
+&sdhi2 {
+	iommus = <&ipmmu_ds1 34>;
+};
+
+/* ============================ Xen pass through section =====================*/
+
+/*&adsp			{ xen,passthrough; };*/
```
Comment (andr2000) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-4x2g-ab-dom0.dts:299]: Please remove dead code

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-4x2g-ab-domd.dts [hunk 2]
```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Device Tree Source for the H3ULCB Aosbox (CCPF Starter kit) board
+ * running XEN hypervisor
+ *
+ * Copyright (C) 2021 Renesas Electronics Corp.
```
Comment (andr2000) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-4x2g-ab-domd.dts:6]: Here and elsewhere: do we need to put EPAM's copyright as well?

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/ulcb-ab.dtsi [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+				remote-endpoint = <&hdmi1_con>;
+			};
+		};
+	};
+};
+
+/ {
+	hdmi1-out {
+		compatible = "hdmi-connector";
+        // TODO Do we need a label for hdmi0?
```
Comment (andr2000) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/ulcb-ab.dtsi:128]: This TODO needs to be resolved

**Review by andr2000 — APPROVED** [review]
With small fixes: `Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`

**Comment by iusyk** [comment]
Reviewed-by: Ihor Usyk <ihor_usyk@epam.com>

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-4x2g-ab-dom0.dts (hunk 1 above)
Comment (varder) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-4x2g-ab-dom0.dts]: fixed

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-4x2g-ab-domd.dts (hunk 2 above)
Comment (varder) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-h3ulcb-4x2g-ab-domd.dts:6]: added copyright

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/ulcb-ab.dtsi (hunk 3 above)
Comment (varder) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/ulcb-ab.dtsi]: removed unused label and checked

**Pull request merged**